test/unit/rails_sqli_test.rb

```diff
@@ -7,13 +7,13 @@ def test_action_controller_params_are_escaped
     # This should fix # https://groups.google.com/forum/#!topic/rubyonrails-security/8CVoclw-Xkk
     # No error should be raised on that and the SQL
 
-    CMS::Section.find_by(params)
-    sanitized = ActiveRecord::Base.sanitize params['system_name'].to_s
-    assert_match sanitized, CMS::Section.where(params).to_sql
+    CMS::Section.find_by_system_name(params['system_name'].to_s)
+    sanitized = ActiveRecord::Base.connection.quote params['system_name'].to_s
+    assert_match sanitized, CMS::Section.where(system_name: params['system_name'].to_s).to_sql
   end
 
   def test_quoting_acton_controller_parameters
     params = ActionController::Parameters.new('system_name' => {'0' => 'foo'})
-    master_account.sections.find_by_system_name(params[:system_name])
+    master_account.sections.find_by_system_name(params[:system_name].to_s)
```

Comment:
This is not what we want ;)
What I meant is if you remove the initializer, does it pass? meaning it was already fixed by rails so we do not need it anymore.

Reply:
@hallelujah No, if we remove initializer then it fails and so we need to apply this workarounds. if we keep initializer file no workarounds are needed I think.

As I mentioned earlier my first commit fixed the issue in which we are not removing initialize filer. I can remove the second commit and this depreciation is fixed.

Reply:
2 subjects:

1. So removing the initializer will open a vulnerability that was not fixed in Rails, so keep it.
2. The point here was that your change is not testing anything...

Reply:
@hallelujah for your 2 point, if we are keeping initializer, no changes are needed in test. All changes will be reverted and test remains same as it was before.